### PR TITLE
Add recipes for http3 and requests-async

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: recipes/http3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - rfc3986 ==1.*
+
+test:
+  imports:
+    - http3
+    - http3.dispatch
+
+about:
+  home: https://github.com/encode/http3
+  license: BSD
```
Comment (marcelotrevisani) [inline recipes/http3/meta.yaml]: I think that should be 
```BSD 3-Clause```

File: recipes/requests-async/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - requests ==2.*
+
+test:
+  imports:
+    - requests_async
+
+about:
+  home: https://github.com/encode/requests-async
+  license: BSD
+  license_family: BSD
```
Comment (marcelotrevisani) [inline recipes/requests-async/meta.yaml]: I think here should be ``Apache``

File: recipes/requests-async/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - python >=3.6
+    - requests ==2.*
+
+test:
+  imports:
+    - requests_async
+
+about:
+  home: https://github.com/encode/requests-async
+  license: BSD
```
Comment (marcelotrevisani) [inline recipes/requests-async/meta.yaml]: Could you please double check the license? I think that should be ``Apache-2.0``

Comment (marcelotrevisani) [reply recipes/requests-async/meta.yaml]: https://github.com/encode/requests-async/blob/master/LICENSE.md

Comment (nicoddemus) [reply recipes/requests-async/meta.yaml]: Indeed, the metadata on PyPI is incorrect.

Comment (nicoddemus) [reply recipes/requests-async/meta.yaml]: https://github.com/encode/requests-async/pull/56

**Comment by marcelotrevisani** [comment]
Just another thing but I think that is not necessary to change it now, you can change it in the up coming versions
nowadays conda-forge recipes are using this command ``{{ PYTHON }} -m pip install . -vv`` to install the packages

**Comment by marcelotrevisani** [comment]
LGTM!

**Comment by nicoddemus** [comment]
Good to know, thanks!

**Pull request merged**